### PR TITLE
Added README for collector template

### PR DESCRIPTION
**Pull request by sameerbag**
Added crowdstrike readme for collector template

### CODE REVIEW

File: collectors/crowdstrike/cfn/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+# Verification 
+- Go to the newly created lambda function and select Monitoring. 
+There should not be any errors on Invocation errors.
+- Go to the CloudWatch Logs and select `/aws/lambda/<function-name>` Log 
+Group. Open the newest Log Stream and verify it doesn't contain any errors.
+
+
+# Known issues
```
Comment (rakhimundhada15) [inline collectors/crowdstrike/cfn/README.md:49]: If there is no issues ,please remove this heading Known issues.

**Review by rakhimundhada15 — CHANGES_REQUESTED** [review]

File: collectors/crowdstrike/cfn/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   - `AlertlogicCustomerId` - Optional. Alert Logic customer ID which collected data should be reported for. If not set customer ID is derived from AIMs tokens
+   - `AlertlogicSecretKey` - `secret_key` returned from AIMS
+   - `CrowdstrikeAPINames` - Define API names. Please pass JSON formatted list. Possible values are ["Incident", "Detection"]
+   - `CrowdstrikeClientId` - Crowdstrike Client ID 
+   - `CrowdstrikeEndpoint` - Crowdstrike Endpoint
+   - `CrowdstrikeSecret` - Crowdstrike Secret 
+   - `CollectionStartTs` - example `2019-11-21T16:00:00Z` Timestamp when log collection starts
+   - `CollectorId` - default `none` Optional. A collector UUID if known.
+   - `PackagesBucketPrefix` - example `alertlogic-collectors` S3 bucket name prefix where collector packages are located.
+   - `PawsCollectorTypeName` - use `crowdstrike`
```
Comment (kkuzmin) [inline collectors/crowdstrike/cfn/README.md:37]: I think we don't have this param in crowdstrike template. Please double -check

File: collectors/crowdstrike/cfn/README.md (hunk 1 above)
Comment (sameerbag) [reply collectors/crowdstrike/cfn/README.md]: Done

File: collectors/crowdstrike/cfn/README.md (hunk 2 above)
Comment (sameerbag) [reply collectors/crowdstrike/cfn/README.md]: Done

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**